mscore/preferences.cpp

```diff
@@ -123,6 +123,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_APP_TELEMETRY_ALLOWED,                           new BoolPreference(false, false)},
             {PREF_APP_STARTUP_TELEMETRY_ACCESS_REQUESTED,          new StringPreference("", false)},
             {PREF_APP_BACKUP_GENERATE_BACKUP,                      new BoolPreference(true)},
+            {PREF_APP_BACKUP_PATH,                                 new StringPreference(".MuseScoreBackup")},
```

Comment:
I think this new feature should be opt-in, and so the default value of the `PREF_APP_BACKUP_SUBFOLDER` preference should be an empty string.

Reply:
I personally don't see why backup files by not being grouped in a backup dir. I have been annoyed by all these backups stacking up.